inc/utilities.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Utility functions for the Annual Report plugin.
+*/
+
+namespace WMF\Reports\Utilities;
+
+/**
+ * Convert a color slug to a CSS variable.
+ *
+ * @param string $slug The color slug.
+ * @return string The CSS variable.
+ */
+function color_slug_to_css_variable( string $slug ): string {
+	// If the slug ends with a number, add a hyphen before it so it matches the CSS variable name.
+	$fallback = preg_replace( '/([a-zA-Z])(\d)/', '$1-$2', $slug );
+
+	return "var(--wp--preset--color--$slug, var(--wp--preset--color--$fallback));";
```

Comment:
I feel like there's core functionality we should be able to lean on here, but got as far as `getValueFromVariable` in Gutenberg's global-styles `utils.js` and couldn't find anything specifically useful _here_.

I do think it would be useful to have an example in the doc block of a $slug and its corresponding actual variable value, this smells like magic string magic and a little extra documentation wouldn't hurt.